containers/foreman.yml

```diff
@@ -7,70 +7,24 @@
     cluster_up: false
     pulp_worker_count: 2
     registry: quay.io/foreman
-    foreman_providers: false
```

Comment:
Does this PR just remove this? To be clear, that's fine with me as it was really just for the POC.

Reply:
Aye, it does. Cleaning up as part of this change.